src/ResourceManager/Profile/Commands.Profile/Context/SaveAzureRMContext.cs

```diff
@@ -26,8 +26,10 @@ namespace Microsoft.Azure.Commands.Profile
     /// <summary>
     /// Saves Microsoft Azure profile.
     /// </summary>
-    [Cmdlet(VerbsData.Save, "AzureRmProfile", SupportsShouldProcess = true), OutputType(typeof(PSAzureProfile))]
-    public class SaveAzureRMProfileCommand : AzureRMCmdlet
+    [Cmdlet(VerbsData.Save, "AzureRmContext", SupportsShouldProcess = true), OutputType(typeof(PSAzureProfile))]
+    [Alias("Save-AzureRmProfile")]
+    [Obsolete("Save-AzureRmProfile will be renamed to Save-AzureRmContext in the next release.", false)]
+    public class SaveAzureRMContextCommand : AzureRMCmdlet
```

Comment:
@markcowl I was just wondering about static analyzer.
But my main concern was class name changes, I know its not a breaking change, but tests will be broken like ours?